pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonControllerBuilder.java

```diff
@@ -77,6 +77,10 @@
   private final List<Runnable> shutdownActions = new ArrayList<>();
   private RocksDbConfiguration rocksdDbConfiguration;
 
+  /* * */
+  protected Integer pendingTransactionRetentionPeriod;
```

Comment:
Again, adjacent to maxPendingTransactions.  Also, needs a unit or needs to go to Duration.  One of those.